lambda/custom/index.js

```diff
@@ -386,20 +532,102 @@ const UnhandledIntent = {
     const sessionAttributes = handlerInput.attributesManager.getSessionAttributes();
     const requestAttributes = handlerInput.attributesManager.getRequestAttributes();
     if (Object.keys(sessionAttributes).length === 0) {
-      const speechOutput = requestAttributes.t('START_UNHANDLED');
+      let speechOutput = requestAttributes.t('START_UNHANDLED');
+      let repromptText = speechOutput
+      if (supportsDisplay(handlerInput)) {
+        MAIN_DATA_SOURCE.phrase.teaser = speechOutput;
+        MAIN_DATA_SOURCE.phrase.properties.phraseSI = '<speak>' + speechOutput + '</speak>';
+        MAIN_DATA_SOURCE.phrase.reprompt = speechOutput;
+        MAIN_DATA_SOURCE.nextPhrase.teaser = 'none';
+        speechOutput = "";
```

Comment:
You are removing the voice output if there is APL? Why?

Reply:
It will happen only for APL option since the single/two speak items are using this in the transformer instead. This is a way to handle it correctly being backward compatible